src/screens/WasteCollectionScreen.js

```diff
@@ -182,9 +186,15 @@ export const WasteCollectionScreen = ({ navigation }) => {
         })
         ?.filter((address) => (wasteAddressesTwoStep ? address.city === inputValueCity : true));
 
-      return _sortBy(streets, 'street').slice(0, wasteAddressesStreetCount);
+      const sortedList = _sortBy(streets, 'street');
```

Comment:
i would suggest `sortedStreets` here instead of `list` which does not say anything about the object.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/9e1dd698a77e4699502272048afcdf6b63799d17